Source/Core/DolphinQt2/RenderWidget.h

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include <QEvent>
+#include <QTimer>
```

Comment:
To avoid pulling in too many headers, QTimer can be forward declared here.
i.e. `class QTimer;` in the header, and the `#include` in the cpp file.